.github/workflows/nightly.yml

```diff
@@ -46,46 +37,36 @@ jobs:
         with:
           ref: ${{ inputs.git-ref || github.ref_name }}
 
-      - name: Install release-plz
-        run: cargo install --force --locked --version 0.3.30 release-plz
-
       - name:  Prepare nightly crate
         run: |
-          # Replace the crate name
-          # We are just going to replace the first occurance of surrealdb
-          sed -i "0,/surrealdb/s//surrealdb-nightly/" lib/Cargo.toml
-
           # Get the date and time of the last commit
           date=$(git show --no-patch --format=%ad --date=format:%Y%m%d)
           time=$(git show --no-patch --format=%ad --date=format:%H%M%S)
 
           # Update the version to a nightly one
           # This sets the nightly version to something like `1.20231117.1130416`
           # The 1 at the beginning of the patch number is just so it never starts with zero
+          sed -i "s#^version = \"\([[:digit:]]*\)\..*\"#version = \"\1.${date}.1${time}\"#" Cargo.toml
           sed -i "s#^version = \"\([[:digit:]]*\)\..*\"#version = \"\1.${date}.1${time}\"#" lib/Cargo.toml
 
-          # Get the short commit
-          shortRev=$(git rev-parse --short HEAD)
-
-          # Update the description
-          sed -i "s#^description = \".*\"#description = \"A nightly release of the surrealdb crate based on commit ${shortRev}\"#" lib/Cargo.toml
+          # Bump lock file without updating dependencies and ensure crate still compiles
+          cargo check --no-default-features --features storage-mem
 
           # Commit changes
           # We are not going to push these changes. We do this because
           # some git commands `release-plz` runs do not work in detached state.
           git config --local user.email "actions@users.noreply.github.com"
           git config --local user.name "github-actions[bot]"
-          git commit -am 'Prepare nightly release'
           git checkout -b nightly
```

Comment:
you are not pushing this branch. AFAIK it won't be available to the `release` job

Reply:
Yes I noticed when I ran the workflow 🤦‍♂️ I'm now restructuring the workflows to get around this.